PendingReleaseNotes

```diff
@@ -21,6 +21,8 @@
 * RGW: Adding missing quotes to the ETag values returned by S3 CopyPart,
   PostObject and CompleteMultipartUpload responses.
 * RGW: Added support for S3 GetObjectAttributes.
+ * RGW: Now allow running multiple rgw processes on same host, leveraging the
+  |so_reuseport| socket option allowed by kernel.
```

Comment:
Suggest

* RGW: Facilitate running multiple RGWs processes on same host by leveraging the ``so_reuseport`` socket option.

Reply:
Done